### DISCUSSION TITLE
What's the purpose of category settings?

### DISCUSSION THREAD

**Discussion by msclp** [inventree/InvenTree#7615] [Q&A] [answered]
Hello
I added some parameters to categories (pic rel) but it has no effect on the selected part subcategory. What am I missing here? What's the purpose of this setting?
![image](https://github.com/inventree/InvenTree/assets/20436247/96a67a23-e55d-4506-b6f6-4555da74e7d5)

**Comment by SchrodingersGat** [comment, marked as answer]
These are default template values which are assigned to a *new part* within that category, when it gets created.
